### DISCUSSION TITLE
Welcome to Spectator

### DISCUSSION THREAD

**Discussion by hotmeteor** [hotmeteor/spectator#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jarrodparkes** [comment]
👋 Hey everyone, I'm Jarrod.

Adam (@hotmeteor) and I worked together at Spur, and we collaborated a bit on the idea of "spec driven development". I've continued working on this idea in my new employment at [Aryeo](https://www.aryeo.com), and we are trying to take it even further — using specs to model a versioned API. So far, so good.

This package is a big part of our development cycle because it holds everyone accountable to a single source of truth — the spec. Looking forward to seeing this community grow!

Best,
Jarrod

**Comment by beblife** [comment]
Hello, I'm Laurens.

I discovered this package a while a go when searching for good OpenAPI tooling. Working with API's daily at my current employment I decided to take it for a spin and I was not disappointed! Generally like the core idea this package is trying to achieve.

I'm mainly here to give back to the community by contributing as much as I can. I feel like there needs to be proper tooling available for everyone to enjoy working with OpenAPI using Laravel (and in general). ✌️

**Comment by paulosanda** [comment]
Hi!
I'm having this issue
`#content: "{"exception":"Spectator\\Exceptions\\MissingSpecException","message":"Cannot resolve schema with missing or invalid spec."}"
    #version: "1.1"
    #statusCode: 500
    #statusText: "Internal Server Error"
    #charset: null
    +original: array:2 [
      "exception" => "Spectator\Exceptions\MissingSpecException"
      "message" => "Cannot resolve schema with missing or invalid spec."
`
In my test file i'm using:
`public function setUp(): void
    {
        parent::setUp();

        Spectator::using(api-docs.json);
    }`
